languoids/tree/sino1245/sini1245/sout2740/ping1244/ping1244.ini

```diff
@@ -1,31 +0,0 @@
-# -*- coding: utf-8 -*-
-[core]
-name = Pinghua
-glottocode = ping1244
-hid = NOCODE_Pinghua
-level = language
-latitude = 27.6725
-longitude = 112.101667
-macroareas = 
-	Eurasia
-countries = 
-	China (CN)
-
-[classification]
```

Comment:
Do we need to move this information somewhere else? I.e. is there a new languoid for which this info is appropriate?